detox/src/devices/Device.js

```diff
@@ -42,6 +43,16 @@ class Device {
     await this.client.sendUserNotification(params);
   }
 
+  async setOrientation(orientation) {
```

Comment:
`Device.js` is platform agnostic, it's a base class (will be the base class for Android devices as well). This is a simulator specific logic, hence needs to be in `Simulator.js`